src/__tests__/exceptions.test.js

```diff
@@ -83,7 +83,7 @@ describe('Handling of ill-behaved functions', () => {
       7.807, -3.74, 21.119, 2.382, 4.269, 41.57, 73.401, 98.535, 97.059, 92.147,
     ],
   };
-  it('Should stop and return parameterError=NaN if function evaluates to NaN after starting', () => {
+  it('Should stop and return initialValues if function evaluates to NaN after starting', () => {
```

Comment:
Why remove this test case instead of adding a new one?

Reply:
This test case `Nan check` is not valid anymore given the change I made. We calculate the first `error` outside of the iterations and the error is a valid number as shown in the assertion.